src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -0,0 +1,87 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+
+public class Lights {
+    // Docs -
+    // https://docs.wpilib.org/en/latest/docs/software/actuators/addressable-leds.html
+    private AddressableLED ledStrip;
+    private AddressableLEDBuffer ledBuffer;
+
+    private final RGBValues RED = new RGBValues(255, 0, 0);
+    private final RGBValues BLUE = new RGBValues(0, 0, 139);
+    private final RGBValues LIME_GREEN = new RGBValues(0, 255, 0);
+
+    public Lights(int pwmPort, int numLeds) {
+        ledStrip = new AddressableLED(pwmPort);
+        ledBuffer = new AddressableLEDBuffer(numLeds);
+        ledStrip.setLength(ledBuffer.getLength());
+        ledStrip.setData(ledBuffer);
+        ledStrip.start();
+    }
+
+    public void allBlue() {
+        this.setColor(BLUE);
+    }
+
+    public void allLimeGreen() {
+        this.setColor(LIME_GREEN);
+    }
+
+    private void setColor(RGBValues color) {
+        for (int x = 0; x < ledBuffer.getLength(); x++) {
```

Comment:
Should use ++x instead of x++.
https://stackoverflow.com/questions/1094872/is-there-a-difference-between-x-and-x-in-java

Reply:
The two forms are identical in a for loop. 

1 First, initialization is performed (x=0)

2 the check is performed (x < ledBuffer.getLength())

3 the code in the loop is executed.

4 the value is incremented

5 Repeat steps 2 - 4

This is the reason why, there is no difference between x++ and ++x in the for loop

Reply:
https://stackoverflow.com/questions/1918196/why-doesnt-changing-the-pre-to-the-post-increment-at-the-iteration-part-of-a-fo

Reply:
There is no programmaticly observable difference between the two in this usage, however, it's extra work for the compiler to convert it (the machine code will want to prefix, so the compiler searches out usages of postfixing, and if it's not required, it converts it), and further, if someone wanted to print the incremented/decremented value while incrementing/decrementing (which is a much more common usage than printing the original value), they would need to prefix.  

The Java Standard does not take a side, but the standard everywhere I've worked is to prefix.